wordpress.php

```diff
@@ -38,6 +38,19 @@
     run('curl {{ url }}/kinsta-clear-cache-all/');
 });
 
+task('wordpress:set-wp-home', function () {
+    // add WP_HOME to environment file
+    $file = realpath($_SERVER['DOCUMENT_ROOT']) . '/config/environments/' . currentHost()->getAlias() . '.php';
+    $lines = file($file);
+    $newLines = [];
+    foreach ($lines as $line) {
+        if (strpos($line, "Config::define('WP_HOME'") === false) {
+            $newLines[] = $line;
+        }
+    }
+    $newLines[] = 'Config::define(\'WP_HOME\', \'' . get('url', 'https://') . '\');';
```

Comment:
this is quite difficult to read because of the escaping. compare to eg.

```suggestion
    $newLines[] = sprintf("Config::define('WP_HOME', '%s');", get('url', 'https://'));
```